lib/bitcrowd_ecto/changeset.ex

```diff
@@ -280,4 +280,37 @@ defmodule BitcrowdEcto.Changeset do
       changeset
     end
   end
+
+  @hex_color_regex ~r/^#[0-9A-Fa-f]{6}$/
+
+  @doc """
+  Validates a changeset field with hexadecimal color format
+  """
+  def validate_hex_color(changeset, hex_color_field) do
+    validate_format(changeset, hex_color_field, @hex_color_regex)
+  end
+
+  @doc """
+  validates a date field in the changeset is after the reference date
+  passed as `ref_date`
+  """
+  def validate_date_after(changeset, date_field, ref_date) do
```

Comment:
```suggestion
  @doc since: "0.11.0"
  @spec validate_date_after(Ecto.Changeset.t(), atom, Date.t()) :: Ecto.Changeset.t()
  def validate_date_after(changeset, date_field, ref_date) do
```